jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -160,8 +161,13 @@ def process(self, input_data):
                 bbox = wcs_bbox_from_shape(result.data.shape)
                 if result_exptype == "nis_wfss":
                     imaging_func = niriss_imaging
-                else:
+                elif result_exptype == "nrc_wfss":
                     imaging_func = nircam_imaging
+                elif result_exptype == "mir_wfss":
+                    imaging_func = miri_imaging
+                    self.sip_max_inv_pix_error = self.sip_max_inv_pix_error * 2
+                else:
+                    log.warning(f" EXP Type is not of a WFSS mode {result_exptype}")
```

Comment:
Because the `if` statement on line 132 checks the wfss types, I don't think it's possible for this `else` to be hit and it could be removed.